commonTypes.h

```diff
@@ -145,7 +145,9 @@ struct GameConfiguration {
     bool show_stockpiles;
     bool show_designations;
     bool show_osd;
+    bool show_announcements;
     bool show_keybinds;
+    bool closeOnEsc;
```

Comment:
don't you need to initialize these to default values somewhere? What if the player doesn't have these tokens in their init file? I see some initialization in the `stonesense_thread` function. Perhaps you need to add a few lines there.